apps/sensenet/src/components/appbar/desktop-nav-menu.tsx

```diff
@@ -65,44 +92,66 @@ export const DesktopNavMenu: React.FunctionComponent = () => {
   const globalClasses = useGlobalStyles()
   const theme = useTheme()
   const service = injector.getInstance(PersonalSettings)
-  const { openDialog } = useDialog()
-  const [open, setOpen] = useState(false)
-  const anchorRef = useRef<HTMLButtonElement>(null)
   const currentUser = useCurrentUser()
   const repo = useRepository()
   const localization = useLocalization()
+  const { openDialog } = useDialog()
+  const [openLogoutMenu, setOpenLogoutMenu] = useState(false)
```

Comment:
It is not about logout, it is about the user, so userMenu would be a much better name.
(replace logoutMenu in localization too)